public/configurations/queries/geomap-graph.json

```diff
@@ -0,0 +1,10 @@
+{
+    "id":"test-geomap-graph",
```

Comment:
id should be name of the file.

I would change the name to aar-enterprise-geomap.json